### DISCUSSION TITLE
How to replace Field default Template in the custom module, please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#14629] [Q&A]
How to replace OrchardCMS Field default Template in the custom module, please?

**Comment by Piedone** [comment]
Check out https://docs.orchardcore.net/en/latest/docs/reference/modules/Templates/#content-field-templates.

**Reply by effapp** [reply]
Hi @piedone :

It is invalid to put custom OrchardCMS field template in the custom module project, but it is valid to put it in  web app project.

**Reply by Piedone** [reply]
You can put them in either, as well as into a theme project. What did you try, exactly?

BTW just to confirm: Are you talking about the .NET Framework/ASP.NET edition called Orchard CMS or Orchard 1, or the .NET/ASP.NET Core edition called Orchard Core? (This discussion board is for the latter.)
